lib/usdUfe/ufe/UsdContextOps.cpp

```diff
@@ -306,6 +310,14 @@ Ufe::ContextOps::Items UsdContextOps::getItems(const Ufe::ContextOps::ItemPath&
                     items.emplace_back(kUSDToggleVisibilityItem, label);
                 }
             }
+
+            // Set as Default Prim:
+            // If the prim is a root prim, add set default prim
+            if (prim().GetPath().IsRootPrimPath()) {
+                items.emplace_back(kUSDSetAsDefaultPrim, kUSDSetAsDefaultPrim);
+                items.emplace_back(kUSDClearDefaultPrim, kUSDClearDefaultPrim);
```

Comment:
The spec in EMSUSD-105 says that clear should only be provided if the prim is the current default one.